src/style/index.css

```diff
@@ -1,5 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&family=Roboto+Slab:wght@200;300;700&display=swap');
+
 body {
+  width: 100%;
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: 'Montserrat', sans-serif;
+  background-color: lightgray;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_3_6_2022_3-02-48 AM](https://user-images.githubusercontent.com/45217364/156904846-50b26842-3547-4fbf-ba3f-49df5a7e7707.png) |

  - As I mentioned in the previous review, the app is not fullscreen, it is more like a card in the center of the page.
  - `Add new book` Content should be horizontal and vertically aligned with the other contents
  - As I mentioned in the previous review, try to use the exact color as mockup design. for instance, the background color you used is too dark. try to lighten it a little.
  - As the project requires, please try to make it look more like the provided mockup design as much as possible.
  ![scrnli_3_6_2022_3-11-17 AM](https://user-images.githubusercontent.com/45217364/156905084-7826db40-c761-4884-b1ac-12da989faffa.png)